runtime/queries/nix/indents.scm

```diff
@@ -1,18 +1,40 @@
+
```

Comment:
I think these changes can mostly be reverted. Helix only supports `@indent` and `@outdent` plus some sub-scopes on `@indent` rather than `@branch` or `@ignore`